### ISSUE TITLE
'MODULE_NOT_FOUND'

### ISSUE THREAD

**Issue by flyingstarlai** [saltyshiomix/nextron#44]
```
npm install --global nextron@latest
nextron init excel-trans --template with-typescript-material-ui
cd excel-trans
yarn install
```

```
`➜  excel-trans yarn dev
yarn run v1.13.0
$ nextron
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:8888 ...
[ error ] ./pages/_app.tsx
Module not found: Can't resolve '../components/theme' in '/Developments/Nextron/excel-trans/renderer/pages'
[ wait ]  compiling ...
[ error ] ./pages/_app.tsx
Module not found: Can't resolve '../components/theme' in '/Developments/Nextron/excel-trans/renderer/pages'
`
```

**Comment by saltyshiomix** [comment]
@flyingstarlai 

Thank you for your reporting the issue!
I'll fix this as soon as possible.

**Comment by saltyshiomix** [comment]
@flyingstarlai 

Fixed the bug as of `nextron@^5.5.0`.
Could you try it?

```bash
$ npm i -g nextron@latest

$ nextron init excel-trans --template with-typescript-material-ui
$ cd excel-trans
$ yarn && yarn dev
```

**Comment by flyingstarlai** [comment]
thans for the respond.. it works great now
